bin/phantomjs-runner.js

```diff
@@ -0,0 +1,56 @@
+var page = require('webpage').create();
+var system = require('system');
+var args = system.args;
+
+console.log("running browser tests with cookie value", args[1]);
+
+phantom.addCookie({
+  'name': 'ring-session',
+  'value': args[1],
+  'domain': 'localhost'
+});
+
+global.testsSuccessful = undefined;
+var url = 'http://localhost:8350/lomake-editori/test.html';
+var resultPrefix = '*** TEST';
+var successMsg = ' SUCCESS';
+var failMsg = ' FAIL';
+
+function startsWith(haystack, needle) {
+  return haystack.substring(0, needle.length) === needle
+}
+
+page.onConsoleMessage = function (message) {
+  console.log(message);
+  if (startsWith(message, resultPrefix)) {
```

Comment:
This is extra terrible. Let me know if you figure out a proper way to run the tests in a blocking manner...